webpack.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   },
```

Comment:
- [ ] I would recommend you set your **mode** attribute in this file to complete the configuration.

  ![ackon-mode](https://user-images.githubusercontent.com/36542986/155500265-39fc307b-e542-4b6c-a0e0-131560adf14b.jpg)

Reply:
Thanks for the tip